src/overlays/actors/ovl_En_Test7/z_en_test7.c

```diff
@@ -32,80 +45,945 @@ const ActorInit En_Test7_InitVars = {
+
+    camera->fov = ((this->unk_1E78 - camera->fov) * arg2) + camera->fov;
+}
+
+void func_80AF2808(EnTest7* this, GlobalContext* globalCtx, f32 arg2) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    player->actor.shape.rot.y += 0x2EE0;
+    player->actor.scale.x = ((0.0f - this->unk_1E90) * arg2) + this->unk_1E90;
+    player->actor.scale.z = ((0.0f - this->unk_1E94) * arg2) + this->unk_1E94;
+}
+
+void func_80AF2854(EnTest7* this, GlobalContext* globalCtx) {
+    f32 temp;
+
+    if ((this->unk_1E54 >= 12) && (this->unk_1E54 < 31)) {
+        temp = (this->unk_1E54 - 12) / 18.0f;
+        func_80AF24D8(this, globalCtx, temp);
+    } else if ((this->unk_1E54 >= 79) && (this->unk_1E54 < 96)) {
+        temp = (this->unk_1E54 - 79) / 16.000000001f;
+        func_80AF2654(this, globalCtx, temp);
+    }
+
+    if ((this->unk_1E54 >= 42) && (this->unk_1E54 < 69)) {
+        temp = (this->unk_1E54 - 42) / 26.0f;
+        func_80AF2808(this, globalCtx, temp);
+    }
+}
+
+void func_80AF2938(EnTest7* this, GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    this->unk_1E98 = player->actor.draw;
+    player->actor.draw = NULL;
+    player->stateFlags2 |= 0x20000000;
+    this->unk_144 |= 2;
+    this->unk_148.unk_04 = 30.0f;
+    if (globalCtx->roomCtx.currRoom.unk3 != 1) {
+        func_80AF082C(this, func_80AF2AE8);
+    } else {
+        func_80AF082C(this, func_80AF2EC8);
+    }
+}
+
+void func_80AF29C0(EnTest7* this, GlobalContext* globalCtx) {
+    static s16 D_80AF3450[2] = { 0, 0x31C7 };
+    static f32 D_80AF3454 = 3500.0f;
+    s32 pad;
+    Player* player = GET_PLAYER(globalCtx);
+    Vec3f* pos = &player->actor.world.pos;
+    Camera* temp_s0 = Play_GetCamera(globalCtx, ActorCutscene_GetCurrentCamera(globalCtx->unk_1879C[8]));
+
+    temp_s0->at.x = ((D_80AF3454 * Math_SinS(D_80AF3450[0]) * Math_CosS(D_80AF3450[1]))) + pos->x;
+    temp_s0->at.y = (Math_SinS(D_80AF3450[1]) * D_80AF3454) + pos->y;
+    temp_s0->at.z = ((D_80AF3454 * Math_CosS(D_80AF3450[0])) * Math_CosS(D_80AF3450[1])) + pos->z;
+
+    this->actor.world.pos.x = temp_s0->at.x;
+    this->actor.world.pos.y = temp_s0->at.y - 40.0f;
+    this->actor.world.pos.z = temp_s0->at.z;
+}
+
+void func_80AF2AE8(EnTest7* this, GlobalContext* globalCtx) {
+    Camera* camera;
+
+    if (!ActorCutscene_GetCanPlayNext(globalCtx->unk_1879C[8])) {
+        ActorCutscene_SetIntentToPlay(globalCtx->unk_1879C[8]);
+        return;
+    }
+
+    ActorCutscene_Start(globalCtx->unk_1879C[8], NULL);
+    func_80AF082C(this, func_80AF2C48);
+
+    camera = Play_GetCamera(globalCtx, ActorCutscene_GetCurrentCamera(globalCtx->unk_1879C[8]));
+    this->unk_1E60 = camera->eye;
+    this->unk_1E6C = camera->at;
+
+    func_80AF29C0(this, globalCtx);
+}
+
+void func_80AF2BAC(EnTest7* this, GlobalContext* globalCtx, Vec3f* arg2, f32 arg3) {
+    f32 x, y, z;
+    Camera* camera = Play_GetCamera(globalCtx, ActorCutscene_GetCurrentCamera(globalCtx->unk_1879C[8]));
```

Comment:
```suggestion
    f32 x;
    f32 y;
    f32 z;
    Camera* camera = Play_GetCamera(globalCtx, ActorCutscene_GetCurrentCamera(globalCtx->unk_1879C[8]));
```
Try to avoid same-line declarations